src/pepr/logger.ts

```diff
@@ -0,0 +1,15 @@
+import { Log } from "pepr";
+
+export function childLog(subproject: string) {
+  const childLog = Log.child({ subproject: subproject });
```

Comment:
Not in love with `subproject`, maybe `submodule` or `component` instead?

Reply:
changing to component